src/app/services/platform.service.ts

```diff
@@ -19,4 +19,20 @@ export class PlatformService {
   public save(platform: Platform) {
     return this.http.post<Platform>(this.platformsUrl, platform);
   }
+
+  public findByScoreAsc(): Observable<Platform[]>{
```

Comment:
Great way to implement the sorting mechanisms. The naming is questionable though